product_docs/docs/migration_toolkit/55/03_migration_methodology.mdx

```diff
@@ -1,54 +1,52 @@
 
-2.  Identify potential migration problems. If it is an Oracle-to-EDB Postgres Advanced Server migration, consult the [EnterpriseDB documentation](/epas/latest/) for complete details about the compatibility features supported in EDB Postgres Advanced Server. Consider using EnterpriseDB's migration assessment service to assist in this review.
+2.  Identify potential migration problems. If it is an Oracle-to-EDB Postgres Advanced Server migration, consult the [EnterpriseDB Postgres Advanced Server documentation](/epas/latest/) for complete details about the compatibility features supported in EDB Postgres Advanced Server. Consider using the EnterpriseDB migration assessment service to help with this review.
 
-3.  Prepare the migration environment. Obtain and install the necessary software, and establish connectivity between the servers.
+3.  Prepare the migration environment. Obtain and install the needed software, and establish connectivity between the servers.
 
-4.  If the migration involves a large body of data, consider migrating the schema definition before moving the data. Verify the results of the DDL migration and resolve any problems reported in the migration summary. The [Migration Errors section](09_mtk_errors/#mtk_errors) of this document includes information about resolving migration problems.
+4.  If the migration involves a large body of data, consider migrating the schema definition before moving the data. Verify the results of the DDL migration and resolve any problems reported in the migration summary. [Migration errors](09_mtk_errors/#mtk_errors) includes information about resolving migration problems.
 
-5.  Migrate the data. For small data sets, use Migration Toolkit. If it is an Oracle migration (into EDB Postgres Advanced Server), and the data set is large or if you notice slow data transfer, take advantage of one of the other data movement methods available:
+5.  Migrate the data. For small data sets, use Migration Toolkit. If it's an Oracle migration into EDB Postgres Advanced Server, and the data set is large or if you notice slow data transfer, take advantage of one of the other data movement methods available:
 
     -   Use the EDB Postgres Advanced Server database link feature compatible with Oracle databases.
     -   If your data has BLOB or CLOB data, use the dblink_ora style database links instead of the Oracle style database links.
 
-    Both of these methods use the Oracle Call Interface (OCI) to connect to Oracle. After connecting, use an SQL statement to select the data from the 'linked' Oracle database and insert the data into the EDB Postgres Advanced Server database.
+    Both of these methods use the Oracle call interface (OCI) to connect to Oracle. After connecting, use an SQL statement to select the data from the 'linked' Oracle database and insert the data into the EDB Postgres Advanced Server database.
```

Comment:
Oracle capitalises this term https://www.oracle.com/in/database/technologies/appdev/oci.html.

Reply:
I usually face a similar dilemma on whether to capitalise Azure resources or not. Even Azure docs seems to capitalise them inconsistently in some of their docs. 

For example: in (https://docs.microsoft.com/en-us/azure/private-link/create-private-endpoint-portal) the term private endpoint is inconsistently capitalised.

Reply:
Okay, thanks. Will fix.